sdk/eventhub/Azure.Messaging.EventHubs/design/partition-receiver-proposal.md

```diff
@@ -0,0 +1,451 @@
+
+As a cost-control mechanism, the game and player services are hosted in a densely packed manner; resource utilization for the host environment is intended to run high and is managed cooperatively by the hosted services.  Each service must be mindful of background activities and use of machine and network resources in order ensure that the services are responsive and performant as a whole.
+
+### An interactive streaming media platform can degrade gracefully
+
+As e-sports continue to grow in popularity, many of the high-profile multiplayer games are now sponsoring tournaments and companion applications to watch them streaming.  One such game offers a companion application across different platforms and consumer devices, such as game consoles, PCs, tablets, and mobile phones.  Processing power, memory, and network bandwidth are expected to  differ and the application must degrade gracefully when necessary.
+
+The presentation for matches is a combination of streaming video from the competitors and overlays from the production crew.  Each of these is sent to the application individually, with the client responsible for assembling them for the final presentation.  The video stream is sent via direct socket connection in response to an application request.  Overlay data is communicated passively using an Event Hub with each partition carrying a specific class of information.
+
+To provide the best user experience, the application must prioritize video availability and quality.  The ability for a user to watch a match in real-time, with smooth and even video and audio is essential.  When resources are constrained, the application will compensate by progressively trimming back the experience. To that end, awareness and control over network and resource use is critical for the application to have. 
+
+First, the application will only read higher priority overlay data from the Event Hub partition.  Next, the application will cease reading any overlay data.  Finally, the quality of video will be downgraded.  These mitigations are applied dynamically and the application will endeavor to provide the full experience as constraints allow.
+
+### Manufacturing device upgrade coordination
+
+A large manufacturer makes use of a local Azure Stack instance to provide an Event Hubs service within its onsite network.  The devices on the manufacturing floor are Wi-Fi enabled and make use of events to coordinate with the manfacturing ecosystem.  One important activity that devices coordinate through Event Hubs is firmware upgrades.  It is important that upgrades be performed as efficiently as possible in order to keep manufacturing at optimal levels and not negatively impact production.
```

Comment:
```suggestion
A large manufacturer makes use of a local Azure Stack instance to provide an Event Hubs service within its onsite network.  The devices on the manufacturing floor are Wi-Fi enabled and make use of events to coordinate with the manufacturing ecosystem.  One important activity that devices coordinate through Event Hubs is firmware upgrades.  It is important that upgrades be performed as efficiently as possible in order to keep manufacturing at optimal levels and not negatively impact production.
```

Reply:
Thanks! Going to make the change directly; suggestions throw off my rebase in ways that aren't always easy to fix.